### DISCUSSION TITLE
Export cropped video from centroid inference without labels

### DISCUSSION THREAD

**Discussion by amblypatty** [talmolab/sleap#1522] [Help!]
Hello,

I am wondering how I might be able to generate animal-centered videos from videos of animals moving around in an arena. For my case, it should be easy because I have single-animal videos. Is there a way to export video of the cropped frames that are created in the top-down centroid model to a video file? In particular, I'd like to generate video similar to the fly_leap_test example videos. I understand that these videos are animal-centered and egocentrized, which might be more than what the top-down cropping does, but if it can save the cropped images, then one could potentially add a rotation function and then save those images as frames to a video. I am not experienced in python though, and don't know how to use the developer API to do so. Any open-source/free help on this front would be appreciated.

**Comment by roomrys** [comment]
The is not an out of the box method, but I started a branch and [draft pr](https://github.com/talmolab/sleap/pull/1535) to add a centroid only predictor as step one of this vision:
1. Centroid Only Predictor
2. Crops about centroid
3. Ego-centralize
